Makefile

```
@@ -13,7 +13,7 @@ OTHER_MOUNTS ?=
 
 # base images w/ or w/o CUDA
 ifeq ($(CUDA),n)
-	BASE_IMAGE ?=ubuntu:19.10
+	BASE_IMAGE ?=ubuntu:18.04
 	DEP_TAG_NAME ?=gnu9-mpich314-nocuda
```

I know the dep_tag_name for cuda based on 18.04 seems to say gnu8 for some reason, but gcc --version and gfortran --version tell me it's version 7 after I install them on an 18.04 image. You may want to check whether this suggested tag is accurate.

```suggestion
	DEP_TAG_NAME ?=gnu7-mpich314-nocuda
```

---

I suggest we change the cuda tag to gnu7 as well, as it is also gnu7. The gnu8 might have been what was wanted to match better with Piz Daint, but the image doesn't have it installed at the moment. A different PR might want to install it there, but given we are moving away from running with containers on daint, perhaps it doesn't matter.